mock-echo-app/src/cmr/mock_echo/api/urs.clj

```diff
@@ -1,15 +1,18 @@
 (ns cmr.mock-echo.api.urs
   "Defines routes for mocking URS"
-  (:require [compojure.core :refer :all]
-            [cheshire.core :as json]
-            [clojure.data.xml :as x]
-            [cmr.common.xml :as cx]
-            [cmr.common.mime-types :as mt]
-            [clojure.string :as str]
-            [cmr.common.services.errors :as errors]
-            [cmr.mock-echo.data.urs-db :as urs-db]
-            [clojure.data.codec.base64 :as b64]
-            [ring.util.response :as rsp]))
+  (:require
+   [cheshire.core :as json]
+   [clojure.data.codec.base64 :as b64]
+   [clojure.data.xml :as x]
+   [clojure.string :as str]
+   [cmr.common.config :refer [defconfig]]
+   [cmr.common.mime-types :as mt]
+   [cmr.common.services.errors :as errors]
+   [cmr.common.xml :as cx]
+   [cmr.mock-echo.data.urs-db :as urs-db]
+   [cmr.transmit.config :as transmit-config]
+   [compojure.core :refer :all]
+   [ring.util.response :as rsp]))
 
```

Comment:
We're styling our `ns`es based on [this stuart sierra blogpost](https://stuartsierra.com/2016/clojure-how-to-ns.html), which I think says to do the above